tests/snuba/api/endpoints/test_discover_query.py

```diff
@@ -417,6 +418,83 @@ def test_meta_types(self):
             {'name': 'count', 'type': 'integer'}
         ]
 
+    def test_access_with_event_v2_feature(self):
```

Comment:
How about running all the existing discover tests against both v1 and v2 in the meantime if we're also expecting them to pass?

Reply:
Good idea.